challenger-ng/src/environments/environment.ts

```diff
@@ -5,6 +5,6 @@ export const environment: IEnvironment = {
   production: false,
   unsplash: {
     // provide add your access key here
-    accessKey: ''
+    accessKey: 'lJUgqdEF9WtSOV9KbPQhkwNmQs4h925gR23IyqAj7kc'
```

Comment:
Would you mind removing your access key?

Reply:
Removed, thank you 👍